src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -258,6 +272,9 @@ const AccessibilityRequestDetailPage = () => {
         <UswdsLink href="#CreateAccessibilityRequestNote-NoteText">
```

Comment:
This one shifts focus, but I'm not sure about the link changing. If anything, the hash should be cleaner.

Reply:
What do you mean about link changing?  What do you mean by cleaner hash?  I copied the existing pattern for mine, but can certainly change the name.